vmicore/src/io/grpc/GRPCServer.cpp

```diff
@@ -70,13 +71,13 @@ namespace VmiCore
         (*server)->send_termination_event();
     }
 
-    void GRPCServer::sendErrorEvent(const std::string& message)
+    void GRPCServer::sendErrorEvent(const std::string_view& message)
     {
-        (*server)->send_error_event(message);
+        (*server)->send_error_event(static_cast<std::string>(message));
     }
 
-    void GRPCServer::sendInMemDetectionEvent(const std::string& message)
+    void GRPCServer::sendInMemDetectionEvent(const std::string_view& message)
```

Comment:
Is there a specific reason you did not update the plugin-system/the plugin to the string_view in this PR as well?

Reply:
No